### PR TITLE
docs: guac guide

### PR DESCRIPTION
**Pull request by migmartri**
Guide of how to send SBOMS to guac

refs https://github.com/chainloop-dev/chainloop/issues/209

### CODE REVIEW

File: docs/guides/guac/guac.mdx [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+## Configure Workflow
+
+By default, attestations generated by a [Chainloop Workflow](/getting-started/workflow-definition) will be picked up by this integration.
+In addition to that, any material of type `SBOM_CYCLONEDX_JSON` or `SBOM_SPDX_JSON` defined in your workflow [contract](/reference/operator/contract) will be taken as well.
+
+<CodeBlock language="yaml" title="example.yaml">
+  {ContractYAML}
+</CodeBlock>
+
+So the first step is to make sure you have a workflow that describes and attest Software Bill Of Materials. Learn more on how to to set it up [here](/guides/sbom-management/).
```
Comment (danlishka) [inline docs/guides/guac/guac.mdx:27]: to to -> to

File: docs/guides/guac/guac.mdx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+## Configure Integration
+
+There are two steps involved to enable this integration:
+
+1. Register a Guac integration in your Chainloop account
+2. Attach this integration to your workflow
+
+### Registration
+
+Currently, this integration only supports [Google Cloud Storage](https://cloud.google.com/storage) (GCS) as storage provider with more to come in the future. If you are interested in a specific provider, please [let us know](https://github.com/chainloop-dev/chainloop/issues/new)
```
Comment (danlishka) [inline docs/guides/guac/guac.mdx:40]: a storage provider, with

File: docs/guides/guac/guac.mdx [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+#### Using Google Cloud Storage (GCS)
+
+**Prerequisites**
+
+- A Google Cloud Platform (GCP) account
+- A GCP project with a GCS bucket
+- A GCP service account with write access to the bucket. See [Creating and managing service accounts](https://cloud.google.com/iam/docs/creating-managing-service-accounts) for more information. 
+- Once create, download the service account [JSON credentials file](https://developers.google.com/workspace/guides/create-credentials#create_credentials_for_a_service_account)
```
Comment (danlishka) [inline docs/guides/guac/guac.mdx:49]: Once created,

**Review by danlishka — APPROVED** [review]
LGTM after fixing some typos

**Comment by migmartri** [comment]
Thanks a lot for proof-reading this and catching the typos. Updated!

**Pull request merged**